From 1d4d09d9dff96f46674262534ce1f0e51a8252cb Mon Sep 17 00:00:00 2001
From: Noah Goldstein <goldstein.w.n@gmail.com>
Date: Sun, 7 Aug 2022 22:42:30 +0800
Subject: [PATCH] x86: Fix `#define STRCPY` guard in strcpy-sse2.S

`#ifndef STPCPY` is incorrect for checking if `STRCPY` is already
defined.  It doesn't end up mattering as the whole check is
guarded by `#if IS_IN (libc)` but is incorrect none the less.

Upstream-Status: Submitted [https://sourceware.org/bugzilla/show_bug.cgi?id=29454 https://sourceware.org/pipermail/libc-alpha/2022-August/141289.html]
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 sysdeps/x86_64/multiarch/strcpy-sse2.S | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sysdeps/x86_64/multiarch/strcpy-sse2.S b/sysdeps/x86_64/multiarch/strcpy-sse2.S
index e29b411314..d6b9bae5f8 100644
--- a/sysdeps/x86_64/multiarch/strcpy-sse2.S
+++ b/sysdeps/x86_64/multiarch/strcpy-sse2.S
@@ -22,7 +22,7 @@
 
 # include <sysdep.h>
 
-# ifndef STPCPY
+# ifndef STRCPY
 #  define STRCPY __strcpy_sse2
 # endif
 
